EIPS/eip-5190.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: 5190
+title: Account-bound Finance
+description: A smart contract design pattern in decentralized finance to remove insolvency and arbitrary loss of funds using blockchain technology only. Extension of EIP-4973
+author: Hyungsuk Kang (@hskang9)
+discussions-to: https://ethereum-magicians.org/t/
+type: Standards Track
+category: ERC
+status: Draft
+requires: 721, 1155, 4973
+created: 2022-06-29
+---
+
+## Abstract
+This EIP proposes a form of smart contract design pattern and a new type account abstraction on how one's finance should be managed in web3.0, ensuring transparency of managing investments and protection with self-sovereignty even from its financial operators.
+
+This EIP also extends account-bound token from [EIP-4973](https://eips.ethereum.org/EIPS/eip-4973), but the difference is that it can be transferred to other account for mobility between managing device.
+
+## Motivation
+
+Finance in crypto ecosystem are facing huge trust issue. Smart contracts can be now a proxy of multisig for operating teams, and Insolvencies do exist in that there is no trace of funds written in blockchain. That can create arbitrary loss of funds as happened in official bankruptcy announcement of Celsius and UST depegging and anchor protocol failure. Smart contract should be used for peer-to-peer only, with immutable rules etched so that only one who has the private key has full access of managing his investment as bitcoin promises. 
+
+Illustration TBD
+
+The pattern empowers more self-sovereignty in users, and gives more credentials(obviously with Account-bound token) than locking financial data in operating team's contract.
+
+Illustration TBD
+
+Segregation between investor's fund and operating fee is clearly specified in the smart contract, so investors can ensure safety to arbitrary loss of funds by operating team's control.
```

Comment:
```suggestion
The separation between an investor's fund and the operating fee is clearly specified in the smart contract, so investors can ensure safety from arbitrary loss of funds through the operating team's control.
```

I think this is what you are trying to say. Can you confirm?

Reply:
Ah yes Segregation is definitely not the right term.